modules/ROOT/nav.adoc

```diff
@@ -1,7 +1,8 @@
 include::{partialsdir}/attributes.adoc[]
 
 * xref:getting-started.adoc[Getting Started]
-** xref:getting-started.adoc#procedure[Setting Up]
+** xref:getting-started.adoc#introduction[Introduction]
```

Comment:
do we need this node? 
altho It would be useful for all situations if we leave toc in place, ie click 'Introduction' to skip toc